pom.xml

```diff
@@ -380,28 +383,6 @@ under the License.
 	</dependencyManagement>
 
 	<profiles>
-		<profile>
-			<id>opensearch1</id>
-			<activation>
-				<jdk>[1.8,)</jdk>
-			</activation>
-			<modules>
-				<module>flink-connector-opensearch</module>
-				<module>flink-connector-opensearch-e2e-tests</module>
-				<module>flink-sql-connector-opensearch</module>
-			</modules>
-		</profile>
-		<profile>
-			<id>opensearch2</id>
-			<activation>
-				<jdk>[11,)</jdk>
-			</activation>
-			<modules>
-				<module>flink-connector-opensearch2</module>
-				<module>flink-connector-opensearch2-e2e-tests</module>
-				<module>flink-sql-connector-opensearch2</module>
-			</modules>
-		</profile>
 		<!-- The spotless comes from flink-connector-parent and needs version bump to support JDK-21 -->
 		<profile>
```

Comment:
may be put a comment here that we could enable it after dropping support of 1.20, since there is no issue with jdk11+

Reply:
makes sense, probably need to enable spotless for jdk21 there
i'll submit a PR there